test/feature/test_lightglue_onnx.py

```diff
@@ -76,10 +85,15 @@ def test_exception(self):
         assert "Wrong dtype" in str(e)
 
         # Wrong device
+        wrong_device = torch.device("cpu" if device.type == "cuda" else "cuda")
         with pytest.raises(Exception) as e:
             model(
                 {
-                    "image0": {"keypoints": torch.zeros(1, 5, 2, device="cpu"), "descriptors": desc, "image": image},
+                    "image0": {
+                        "keypoints": torch.zeros(1, 5, 2, device=wrong_device),
```

Comment:
usage question -- should we support non batched inputs ? (and numpy ?) /cc @johnnv1 @ducha-aiki